depthai.py

```diff
@@ -131,6 +147,66 @@ def stream_type(option):
         stream_dict = {'name': stream_name, "max_fps": max_fps}
     return stream_dict
 
+def decode_tracklets(packetData):
```

Comment:
It would be great to add a docstring with packetData format. I'd also suggest naming this in the more pythonic snake_case --> `packet_data`